data/interfaces.txt

```diff
@@ -99,7 +99,7 @@ color "missile dangerous" 1. .1 .1 1.
 color "radar player" .2 1. 0. 0.
 color "radar friendly" .4 .6 1. 0.
 color "radar unfriendly" .8 .8 .4 0.
-color "radar hostile" 1. .6 .4 0.
+color "radar hostile" 1. .3 .2 0.
```

Comment:
seems like an unrelated change?
-> separate PR

Reply:
This is done to ensure that the new icon and the colour for hostile ships on the RADAR match.
Doing it in a separate PR seems like a waste of time.

Reply:
ah, didnt see that, fine then